core/src/test/scala/org/scalarelational/AbstractTableSpec.scala

```diff
@@ -546,30 +546,30 @@ trait AbstractTableSpec extends WordSpec with Matchers {
 trait AbstractTestDatastore extends Datastore {
   object test extends Table("test_table") {
     val id = column[Option[Int]]("id", PrimaryKey, AutoIncrement)
-    val name = column[String]("name", Unique, ColumnLength(1024))
+    val name = column[String]("name", Unique, ColumnLength(50))
     val date = column[Option[Timestamp]]("date")
   }
   object suppliers extends Table("SUPPLIER") {
     val id = column[Option[Int]]("SUP_ID", PrimaryKey, AutoIncrement)
-    val name = column[String]("SUP_NAME")
-    val street = column[String]("STREET")
-    val city = column[String]("CITY")
-    val state = column[String]("STATE")
-    val zip = column[String]("ZIP")
+    val name = column[String]("SUP_NAME", ColumnLength(200))
```

Comment:
We should be able to rely on defaults. If we can't then something is wrong with the implementation.